<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIFrame</class>
 <widget class="QFrame" name="GUIFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rdvDICOM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#minimizeButton
{
    image: url(:/img/minimize-button1.png);
}

#minimizeButton, #maximizeButton
{
    background-color: transparent;
    border-radius: 0px;
}

#minimizeButton:hover, #maximizeButton:hover
{
    border-style: none;
    background-color:  rgba(128, 128, 128, 0.5);
}

#minimizeButton:pressed, #maximizeButton:pressed
{
  	border-style: none;
	background-color: rgba(128, 128, 128, 0.6);
}

#closeButton
 {
    image: url(:/img/close-button1.png);
    background-color: transparent;
    border-radius: 0px;
}

#closeButton:hover {
    image: url(:/img/close-button2.png);
    border-style: none;
    background-color: #e81123;
}

#closeButton:pressed {
    image: url(:/img/close-button1.png);
    border-style: none;
    background-color: #8c0a15;
}

#labelTitle
{
    font-weight: bold;
    color:  white;
}

#GUIFrame
{
	background-color: #121212;
}

#widgetTopBar
{
	background-color: #121212;
}

#icon
{
	margin-top: 5px;
	margin-right:5px;
	margin-bottom:5px;
}

QScrollBar:vertical {
    background: #2f3136;
    width: 5px;
    margin: 0;
}

QScrollBar::handle:vertical {
    background: #888;
}

QScrollBar::handle:hover:vertical {
  background: #555;
}

QScrollBar::add-line:vertical {
    height: 0px;
}

QScrollBar::sub-line:vertical {
    height: 0px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    height: 0px;
    background: none;
}
QMenuBar
{
    background-color: #bb86fc;
    color : white;
}

QMenu
{
    background-color:#2a2a2c;
    color : white;
}

QMenuBar::item:selected,
QMenu::item:selected {
	background-color:  rgba(128, 128, 128, 0.5);
}
</string>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTopBar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="topBarLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QLabel" name="icon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>:/res/icon_small.png</pixmap>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutMenu">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTitle">
          <property name="text">
           <string>rdvDICOM - Visualisation d'images DICOM</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="minimizeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>49</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>49</width>
            <height>30</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Minimize</string>
          </property>
          <property name="iconSize">
           <size>
            <width>34</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="maximizeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>49</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>49</width>
            <height>30</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Maximize</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="iconSize">
           <size>
            <width>34</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>49</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>49</width>
            <height>30</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Close</string>
          </property>
          <property name="iconSize">
           <size>
            <width>34</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetContent" native="true">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minimizeButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIFrame</receiver>
   <slot>onMinimize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>897</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>1031</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maximizeButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIFrame</receiver>
   <slot>onMaximize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>1031</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIFrame</receiver>
   <slot>onClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1005</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>1031</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClose()</slot>
  <slot>onMaximize()</slot>
  <slot>onMinimize()</slot>
 </slots>
</ui>
